manifests/proxydhcp.pp

```diff
@@ -72,12 +72,15 @@
 
     ensure_packages(['grep', 'acl'])
 
-    [$dhcp::dhcp_dir, dirname($foreman_proxy::dhcp_leases)].each |$path| {
+    [$dhcp::dhcp_dir, "${dhcp::dhcp_dir}/dhcpd.conf", dirname($foreman_proxy::dhcp_leases)].each |$path| {
       exec { "Allow ${foreman_proxy::user} to read ${path}":
         command => "setfacl -R -m u:${foreman_proxy::user}:rx ${path}",
```

Comment:
Wouldn't this give the proxy `+x` to the file? it should only be `+r`

Reply:
Yes, that is true

Reply:
However, this is also the case, when we set the permissions recursively on `/etc/dhcp/dhcpcd.conf` :grin:

Reply:
removed it, because from my tests, the require on `dhcp` should be sufficient.